src/main/java/fi/helsinki/opintoni/integration/fileservice/FileServiceRestClient.java

```diff
@@ -121,8 +124,8 @@ private Header getAuthorizationHeader() {
         return new BasicHeader("Authorization", String.format("Bearer %s", token));
     }
 
-    private String getFileUrl(String path) {
-        return String.format("%s?file=%s", filesUrl, path);
+    private String getFileUrl(String path) throws UnsupportedEncodingException {
+        return String.format("%s?file=%s", filesUrl, URLEncoder.encode(path, "UTF-8"));
```

Comment:
Voisi lisätä testin enkoodaukselle jos menee pienellä vaivalla

Reply:
En kyllä löytänyt testejä tuolle Rest toteutukselle, eli kyllä tuossa hieman olisi työtä.

Reply:
Niin tosiaan kun se käyttää nyt testeissä varmaan aina mock-clienttia ja kun tuo logiikka on tuossa rest-clientissa niin testaus menee hankalaksi. Toinen vaihtoehto voisi olla siirtää enkoodaus tapahtumaan ennen kuin kutsu menee file-service-clientille asti, jolloin sen voisi testatakin helpommin. Ok kuitenkin näinkin.

Reply:
Annan olla toistaiseksi nyt näin, niin saadaan tuo uh-file-service käyttöön.